main.js

```diff
@@ -0,0 +1,85 @@
+
+
+console.log("you are here");
+
+
+// create variables needed
+var count;
+var row;
+var container;
+var expAmt;
+var expName;
+var addExpAmt;
+var addUpExp;
+var removeExp = document.querySelector('span');
+var tableEntExp;
+var tableEntName;
+
+
+// this function runs when the button is clicked
+addExpense = function(event){
+  // get the values from the input fields
+
+  addExp = document.createElement('div');
+  expAmt = document.querySelector('.expAmt').value;
+  expName = document.querySelector('.expName').value;
+  console.log('did I get exp amt and name?');
```

Comment:
Good practice checking to see if you got there. you also might want to answer your own question and log those values too.